src/modules/event-stream/handle-stream-event.ts

```diff
@@ -25,5 +27,10 @@ export const handleStreamEvent = (
         });
       }
       break;
+    case EventKind.ShmoBookingUpdate:
+      queryClient.invalidateQueries({
+        queryKey: getActiveShmoBookingQueryKey(acceptLanguage),
+      });
```

Comment:
Do we really need to pass a language here? Invalidating just `"GET_ACTIVE_SHMO_BOOKING_QUERY_KEY"` should invalidate queries using both languages.

Reply:
Ah, probably not? I saw that `acceptLanguage` is required in the query key, and assumed that it couldn't be undefined when used to invalidate either. Pushed a change that makes it optional now, and will test that some.